lib/resque/server/views/working.erb

```diff
@@ -17,10 +17,11 @@
         <% queue = data['queue'] %>
         <td><a class="queue" href="<%=u "/queues/#{queue}" %>"><%= queue %></a></td>
         <td><span class="time"><%= data['run_at'] %></span></td>
+	<% payload = data.key?('payload') ? data['payload'] : {} %>
         <td>
-          <code><%= data['payload'].to_h['class'] %></code>
+          <code><%= payload.key?('class') ? payload['class'] : "—" %></code>
```

Comment:
Thank's for your help, @fw42. This really is my very first try to fix anything in Ruby. 

Actually `data` looks as following:
```
{
  "queue"=>"my_queue",
  "run_at"=>"Thu Jun 30 20:55:35 CEST 2016",
  "payload"=>{
    "class"=>"MyClass",
    "args"=>[["my", "args", "1", 2]],
    "id"=>"9387cec11cb81ae1210cc57739e363fc"
  }
}
``` 
Although `data` may also be `{}` (if there's no job running). The `to_h` was needed, to prevent an error with this empty hash. That's also why I first tried, if the payload actually exists.